build/css/intlTelInput.css

```diff
@@ -1 +1,933 @@
x;background-position:-4691px 0px}.iti-flag.sv{height:12px;background-position:-4713px 0px}.iti-flag.sx{height:14px;background-position:-4735px 0px}.iti-flag.sy{height:14px;background-position:-4757px 0px}.iti-flag.sz{height:14px;background-position:-4779px 0px}.iti-flag.ta{height:10px;background-position:-4801px 0px}.iti-flag.tc{height:10px;background-position:-4823px 0px}.iti-flag.td{height:14px;background-position:-4845px 0px}.iti-flag.tf{height:14px;background-position:-4867px 0px}.iti-flag.tg{height:13px;background-position:-4889px 0px}.iti-flag.th{height:14px;background-position:-4911px 0px}.iti-flag.tj{height:10px;background-position:-4933px 0px}.iti-flag.tk{height:10px;background-position:-4955px 0px}.iti-flag.tl{height:10px;background-position:-4977px 0px}.iti-flag.tm{height:14px;background-position:-4999px 0px}.iti-flag.tn{height:14px;background-position:-5021px 0px}.iti-flag.to{height:10px;background-position:-5043px 0px}.iti-flag.tr{height:14px;background-position:-5065px 0px}.iti-flag.tt{height:12px;background-position:-5087px 0px}.iti-flag.tv{height:10px;background-position:-5109px 0px}.iti-flag.tw{height:14px;background-position:-5131px 0px}.iti-flag.tz{height:14px;background-position:-5153px 0px}.iti-flag.ua{height:14px;background-position:-5175px 0px}.iti-flag.ug{height:14px;background-position:-5197px 0px}.iti-flag.um{height:11px;background-position:-5219px 0px}.iti-flag.us{height:11px;background-position:-5241px 0px}.iti-flag.uy{height:14px;background-position:-5263px 0px}.iti-flag.uz{height:10px;background-position:-5285px 0px}.iti-flag.va{height:15px;background-position:-5307px 0px}.iti-flag.vc{height:14px;background-position:-5324px 0px}.iti-flag.ve{height:14px;background-position:-5346px 0px}.iti-flag.vg{height:10px;background-position:-5368px 0px}.iti-flag.vi{height:14px;background-position:-5390px 0px}.iti-flag.vn{height:14px;background-position:-5412px 0px}.iti-flag.vu{height:12px;background-position:-5434px 0px}.iti-flag.wf{height:14px;background-position:-5456px 0px}.iti-flag.ws{height:10px;background-position:-5478px 0px}.iti-flag.xk{height:15px;background-position:-5500px 0px}.iti-flag.ye{height:14px;background-position:-5522px 0px}.iti-flag.yt{height:14px;background-position:-5544px 0px}.iti-flag.za{height:14px;background-position:-5566px 0px}.iti-flag.zm{height:14px;background-position:-5588px 0px}.iti-flag.zw{height:10px;background-position:-5610px 0px}.iti-flag{width:20px;height:15px;box-shadow:0px 0px 1px 0px #888;background-image:url("../img/flags.png");background-repeat:no-repeat;background-color:#DBDBDB;background-position:20px 0}@media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min--moz-device-pixel-ratio: 2), only screen and (-o-min-device-pixel-ratio: 2 / 1), only screen and (min-device-pixel-ratio: 2), only screen and (min-resolution: 192dpi), only screen and (min-resolution: 2dppx){.iti-flag{background-image:url("../img/flags@2x.png")}}.iti-flag.np{background-color:transparent}
```

Comment:
is it possible not to include auto-generated files in pull-request, it makes it closely to impossible for reviewing